.github/workflows/create_summaries.yml

```diff
@@ -37,18 +39,14 @@ jobs:
       run: |
         python3 convert_results.py --source results/ --output results/summaries/
         python3 convert_results.py --source fuzzing_results/ --output fuzzing_results/summaries/
-    - name: Push new summary
-      env:
-        EMAIL_ADDRESS: ${{ secrets.EMAIL_ADDRESS }}
-        SUMMARY_TOKEN: ${{ secrets.SUMMARY_TOKEN }}
-      run: |
-        git config user.email "$EMAIL_ADDRESS"
-        git config user.name "kaczmarczyck"
-        if [[ `git status --porcelain` ]]; then
-          git add results/summaries/
-          git add fuzzing_results/summaries/
-          git commit -a -m "update summaries"
-          git push origin main
-        else
-          echo 'There were no new reports.'
-        fi
+        git add results/summaries/
+        git add fuzzing_results/summaries/
+    - name: Commit & Push changes
+      uses: actions-js/push@master
```

Comment:
It's forbidden by company policy to point github actions to a branch HEAD.
Please use either a tag (preferred) or a commit id instead

Reply:
Makes sense, good to know!